include/dem/pp_contact_force.h

```diff
@@ -64,8 +66,10 @@ class PPContactForce
       &local_adjacent_particles,
     std::unordered_map<int,
```

Comment:
Again the key of that map should be a particle index.
Everywhere the key should be particle index